ipaserver/install/installutils.py

```diff
@@ -504,11 +574,15 @@ def get_server_ip_address(host_name, unattended, setup_dns, ip_addresses):
             if set(ip_addresses) <= set(ips):
                 ips = ip_addresses
             else:
-                print("Error: the hostname resolves to IP address(es) that are different", file=sys.stderr)
```

Comment:
Same here, please revert

Reply:
done, i've discard all chages of PEP8 that I was making. fixing just in the lines I've changed.